app/workers/update_out_of_date_provider_ids_on_application_choices.rb

```diff
@@ -0,0 +1,32 @@
+class UpdateOutOfDateProviderIdsOnApplicationChoices
+  include Sidekiq::Worker
+
+  def perform
+    return unless out_of_date_application_choices.any?
+
+    update_out_of_date_application_choices
+  end
+
+private
+
+  def out_of_date_application_choices
+    @out_of_date_application_choices ||= FindApplicationChoicesWithOutOfDateProviderIds.call
```

Comment:
what do you think of the convention of using an underscore for these memoized variables? as in `@_out_of_date_application_choices`. i quite like it.

Reply:
done